### ISSUE TITLE
CSS line-height shouldn't have units (px, em, rem. etc.)

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#897]
## What

There are numerous elements where the line-height has a unit (e.g. px, em, rem, etc.). CSS `line-height` should be unites to allow text to resize and reflow correctly. When units are added, it can affect the presentation of text if it is resized by users. This (really old) [article by Eric Meyer](http://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/) on the subject explains more.

Labelling this as accessibility because it can affect some users.
